From mboxrd@z Thu Jan  1 00:00:00 1970
Received: from smtp.kernel.org (aws-us-west-2-korg-mail-1.web.codeaurora.org [10.30.226.201])
	(using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
	(No client certificate requested)
	by smtp.subspace.kernel.org (Postfix) with ESMTPS id B329119F469
	for <linux-kernel@vger.kernel.org>; Thu, 27 Jun 2024 21:02:04 +0000 (UTC)
Authentication-Results: smtp.subspace.kernel.org; arc=none smtp.client-ip=10.30.226.201
ARC-Seal: i=1; a=rsa-sha256; d=subspace.kernel.org; s=arc-20240116;
	t=1719522124; cv=none; b=s4NVapf/lq1T0G/Q2NNKau5FA40v9ogxFJehiFKljlGluyrG4IwJyKQkdXqR/OURtBP0HSErZZwdaXmuVv0rT8SwrpBqOPpSGkwK6Ae/8hs7YRXUE5nMyAoZs/uB6JblAvZqffUHaG3ZRWiQLIyg4QJTawF1ojXw/BJXva3W1Q8=
ARC-Message-Signature: i=1; a=rsa-sha256; d=subspace.kernel.org;
	s=arc-20240116; t=1719522124; c=relaxed/simple;
	bh=5A1xrcYEqkBgQFzZBOgLxlRJaM6yasYiRU8R1ppF8p8=;
	h=Message-ID:Date:From:To:Cc:Subject:References:MIME-Version:
	 Content-Type; b=MFhg8Io6UFReVzW8t0/210YaPgnQyW2ZqSL7ab3IDWqmxqAcgd9EnJIJIELWAcaoAVrp2GTxUFh5QiI1OIr9Ffo0if30VQCO7HRtEDpZ4PQdEhgflszvVK8QVC4+cr/UIx7DoXn4mI+lNs8CG7o1mTwSqxPnqDqujvH8gbq2UHw=
ARC-Authentication-Results: i=1; smtp.subspace.kernel.org; arc=none smtp.client-ip=10.30.226.201
Received: by smtp.kernel.org (Postfix) with ESMTPSA id 6AE9EC4AF09;
	Thu, 27 Jun 2024 21:02:04 +0000 (UTC)
Received: from rostedt by gandalf with local (Exim 4.97)
	(envelope-from <rostedt@goodmis.org>)
	id 1sMwGf-000000031l7-0Doo;
	Thu, 27 Jun 2024 17:02:57 -0400
Message-ID: <20240627210256.912091413@goodmis.org>
User-Agent: quilt/0.68
Date: Thu, 27 Jun 2024 16:50:25 -0400
From: Steven Rostedt <rostedt@goodmis.org>
To: linux-kernel@vger.kernel.org
Cc: Daniel Wagner <dwagner@suse.de>,
 "Luis Claudio R. Goncalves" <lgoncalv@redhat.com>,
 Clark Williams <williams@redhat.com>,
 Daniel Bristot de Oliveira <bristot@kernel.org>
Subject: [for-next][PATCH 3/3] tools: build: use correct lib name for libtracefs feature detection
References: <20240627205022.857212058@goodmis.org>
Precedence: bulk
X-Mailing-List: linux-kernel@vger.kernel.org
List-Id: <linux-kernel.vger.kernel.org>
List-Subscribe: <mailto:linux-kernel+subscribe@vger.kernel.org>
List-Unsubscribe: <mailto:linux-kernel+unsubscribe@vger.kernel.org>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Status: RO
Content-Length: 1448
Lines: 44

From: Daniel Wagner <dwagner@suse.de>

Use libtracefs as package name to lookup the CFLAGS for libtracefs. This
makes it possible to use the distro specific path as include path for
the header file.

Link: https://lkml.kernel.org/r/20240617-rtla-build-v1-1-6882c34678e8@suse.de

Cc: Daniel Bristot de Oliveira <bristot@kernel.org>
Signed-off-by: Daniel Wagner <dwagner@suse.de>
Signed-off-by: Daniel Bristot de Oliveira <bristot@kernel.org>
---
 tools/build/feature/Makefile          | 2 +-
 tools/build/feature/test-libtracefs.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/build/feature/Makefile b/tools/build/feature/Makefile
index ed54cef450f5..489cbed7e82a 100644
--- a/tools/build/feature/Makefile
+++ b/tools/build/feature/Makefile
@@ -213,7 +213,7 @@ $(OUTPUT)test-libtraceevent.bin:
 	$(BUILD) -ltraceevent
 
 $(OUTPUT)test-libtracefs.bin:
-	 $(BUILD) $(shell $(PKG_CONFIG) --cflags libtraceevent 2>/dev/null) -ltracefs
+	 $(BUILD) $(shell $(PKG_CONFIG) --cflags libtracefs 2>/dev/null) -ltracefs
 
 $(OUTPUT)test-libcrypto.bin:
 	$(BUILD) -lcrypto
diff --git a/tools/build/feature/test-libtracefs.c b/tools/build/feature/test-libtracefs.c
index 8eff16c0c10b..29a757a7d848 100644
--- a/tools/build/feature/test-libtracefs.c
+++ b/tools/build/feature/test-libtracefs.c
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0
-#include <tracefs/tracefs.h>
+#include <tracefs.h>
 
 int main(void)
 {
-- 
2.43.0



